x-pack/plugins/observability_solution/apm/public/embeddable/alerting/alerting_failed_transactions_chart/react_embeddable_factory.tsx

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { useEffect } from 'react';
+import { DefaultEmbeddableApi } from '@kbn/embeddable-plugin/public';
+import { ReactEmbeddableFactory } from '@kbn/embeddable-plugin/public';
+import {
+  initializeTitles,
+  useBatchedPublishingSubjects,
+  fetch$,
+} from '@kbn/presentation-publishing';
+import { BehaviorSubject, Subject } from 'rxjs';
+import type { EmbeddableApmAlertingVizProps } from '../types';
+import type { EmbeddableDeps } from '../../types';
+import { ApmEmbeddableContext } from '../../embeddable_context';
+import { APMAlertingFailedTransactionsChart } from './chart';
+
+export const APM_ALERTING_FAILED_TRANSACTIONS_CHART_EMBEDDABLE =
+  'APM_ALERTING_FAILED_TRANSACTIONS_CHART_EMBEDDABLE';
+
+export const getApmAlertingFailedTransactionsChartEmbeddableFactory = (deps: EmbeddableDeps) => {
+  const factory: ReactEmbeddableFactory<
+    EmbeddableApmAlertingVizProps,
+    DefaultEmbeddableApi<EmbeddableApmAlertingVizProps>
+  > = {
+    type: APM_ALERTING_FAILED_TRANSACTIONS_CHART_EMBEDDABLE,
+    deserializeState: (state) => {
+      return state.rawState as EmbeddableApmAlertingVizProps;
+    },
+    buildEmbeddable: async (state, buildApi, uuid, parentApi) => {
+      const { titlesApi, titleComparators, serializeTitles } = initializeTitles(state);
+      const serviceName$ = new BehaviorSubject(state.serviceName);
+      const transactionType$ = new BehaviorSubject(state.transactionType);
+      const transactionName$ = new BehaviorSubject(state.transactionName);
+      const environment$ = new BehaviorSubject(state.environment);
+      const rangeFrom$ = new BehaviorSubject(state.rangeFrom);
+      const rangeTo$ = new BehaviorSubject(state.rangeTo);
+      const rule$ = new BehaviorSubject(state.rule);
+      const alert$ = new BehaviorSubject(state.alert);
+      const kuery$ = new BehaviorSubject(state.kuery);
+      const filters$ = new BehaviorSubject(state.filters);
+      const reload$ = new Subject<boolean>();
```

Comment:
@dominiqueclarke I don't think you need this. This is only needed for the embeddables that are added through the `Add panel` in the `Dashboard` app to make them reactive to the `Refresh` button. I think your embeddables are not listed anywhere in the Add panel menu, right? Then I recommend we remove this from here.